python/tk_mari_projectmanager/project_manager.py

```diff
@@ -109,22 +113,18 @@ def show_new_project_dialog(self):
         """
         self.__new_project_publishes = []
         default_name = self._app.get_setting("default_project_name")
-        
-        # create a background worker that will be responsible for updating
-        # the project name preview as the user enters a name.
-        worker_cb = lambda name: self._generate_new_project_name(name)
-        preview_updater = AsyncWorker(worker_cb)
-        try:
-            preview_updater.start()
 
-            # show modal dialog:            
-            res, new_project_form = self._app.engine.show_modal("New Project", self._app, NewProjectForm, 
-                                                                self._app, self._init_new_project_form,
-                                                                preview_updater, default_name)
-        finally:
-            # wait for the background thread to finish!
```

Comment:
Why did you decide to remove the preview_updater? Wasn't the point that we needed an async updater because things could get slow to update?

Reply:
Why would they be slow? It's just taking a string value and building another string from it and the current context, and then updating a QLabel. I don't see any reason why any of that needs to happen from a separate thread.

Reply:
Is the concern about a path cache miss when getting template fields being slow? That's definitely going to be the minority case, and I don't see why that would dictate needing the complexity of a QThread to run it.

Reply:
Chatted with Manne, and he agrees that we should go ahead with removing the worker thread. If you're happy with this otherwise, @jfboismenu, give it an approval and I'll send it over to QA.